docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -1975,6 +1975,16 @@ public class Endpoint {
 }
 ----
 
+[NOTE]
+====
+You can change the log level of the thrown `WebApplicationException` exceptions by configuring the following property `quarkus.http.WebApplicationException.log.level` like so:
+
+[source, properties]
+----
+quarkus.http.WebApplicationException.log.level=DEBUG
```

Comment:
This property does not exist. The right one I think it's `quarkus.log.category."WebApplicationException".level=DEBUG`

Reply:
Sorry for that, I didn't realize that